app/policies/grant_submission/submission_policy.rb

```diff
@@ -1,17 +1,16 @@
 # frozen_string_literal: true
 
-
 class GrantSubmission::SubmissionPolicy < GrantPolicy
 
   # Can't inherit methods inside scope, so the organization_admin_access?
   # code from AccessPolicy:6 was repeated.
   # https://stackoverflow.com/questions/14739640/ruby-classes-within-classes-or-modules-within-modules
-  attr_reader :user, :grant, :scope
+  attr_reader :user, :grant
 
-  def initialize(context, scope)
+  def initialize(context, record)
```

Comment:
I still don't fully understand this, when to use scope vs record. This would be a good question for Yulia. Git hub is not allowing me to add her name now.